doc/cephfs/index.rst

```diff
@@ -44,15 +44,20 @@ For most deployments of Ceph, setting up a CephFS file system is as simple as:
 
     ceph fs volume create <fs name>
 
-The Ceph `Orchestrator`_  will automatically create and configure MDS for your
-file system if the back-end deployment technology supports it (see
-`Orchestrator deployment table`_). Otherwise, please :doc:`deploy MDS manually
-as needed </cephfs/add-remove-mds>`.
+The Ceph `Orchestrator`_  will automatically create and configure MDS for
+your file system if the back-end deployment technology supports it (see
+`Orchestrator deployment table`_). Otherwise, please `deploy MDS manually
+as needed`_.
```

Comment:
These links don't work in the rendered doc. Please double-check. I thinkt he issue is the absolute reference beginning with `/`. cephfs-shell link is also broken below.

Reply:
Fixed.